### ISSUE TITLE
Can we add some Jazz to this? 

### ISSUE THREAD

**Issue by speckworks** [mahuntington/timer#1]
🙃

How you doing, Matt!???  I've just put in my counter offer to  some contract web dev work in the city.  🤞🏻  I'm getting close to that first coding gig.  Not playing much 'bone, unfortunately.

**Comment by mahuntington** [comment]
That's a funny title because, I'm actually planning on making this a practice timer.  If I get really into it, maybe I'll get the browser to use the mic to stop the timer when it doesn't hear any noise.  That way it only counts time while you're actually playing.  Don't judge my coding by this project.  I will mostly be working on it during meetings.  Expect commits on a biannual basis.

Good luck on that contract!  You graduated at probably the worst time in the last decade, but keep at it.  Things seem to be picking up already, from what I can tell.  Hopefully not much longer for you :)

Yeah, your playing might suffer for next few years, but once you get good enough to freelance, you'll have a lot more free time, and you can pick and choose your gigs!

Keep in touch!

**Comment by speckworks** [comment]
Thanks Matt!! Indeed.  Cool idea.

An old friend of mine from high school has been working on an App to help
Brass Players practice.
https://apps.apple.com/us/app/playtunes-concert-band-time/id1403421290
It's written in Swift by a couple part time coders in the Bay Area.  Def a
Passion Project.  I'd be happy to connect you with Shawn McGinn If you're
interested.  He's a super fun and positive dude.  A high school band
teacher who is slowly bootstrapping this app.

Give a shout any time.

J

**Comment by mahuntington** [comment]
Yeah, can you have him email me at work? matt.huntington@ga.co. I have a class that’s graduating in a few weeks, and I think they’d be happy to help out. Thanks!

**Comment by speckworks** [comment]
cool. . where is work these days?  are these music students or coding students?

**Comment by speckworks** [comment]
ah general assembly..
my bad

**Comment by speckworks** [comment]
I spoke to Shawn, Matt!  He's not quite ready for bootcamp devs yet, but maybe down the line. I'll put you in touch if you like.

**Comment by mahuntington** [comment]
Sounds good!
